Bindings/Java/Matlab/Hopper_Device/BuildHopper.m

```diff
@@ -24,26 +24,29 @@
 % Build a model of a one-leg hopper, with one muscle.
 
 p = inputParser();
-defaultMuscleModel = 'Thelen2003';
+defaultMuscleModel = 'Millard2012Equilibrium';
 defaultMaxIsometricForce = 4000.0;
 defaultMillardTendonParams = [0.049 28.1 0.67 0.5 0.25];
-defaultActivation = [0.0 1.99 2.0 3.89 3.9 4.0;
+defaultExcitation = [0.0 1.99 2.0 3.89 3.9 4.0;
                      0.3 0.3  1.0 1.0  0.1 0.1];
-defaultPrintModel = false; 
+defaultPrintModel = false;
+defaultAdditionalMass = 0;
```

Comment:
I think this makes sense. We could have built-in "slider value to mass/stiffness/maxTension/gain" functions in BuildHopper.m and BuildDevice.m (not hard to move these over). After the workshop, users might want to tweak these functions to augment the challenge for their needs.